core/src/main/java/brooklyn/entity/rebind/dto/MementosGenerators.java

```diff
@@ -68,10 +71,23 @@
 
     private MementosGenerators() {}
     
+    /** @deprecated since 0.7.0 use {@link #newBasicMemento(BrooklynObject)} */
```

Comment:
Why rename from `newMemento(BrooklynObject)` to `newBasicMemento(BrooklynObject)`? Do we have anything but "basic", or are we expecting that in the near future? Personally I prefer `newMemento`, as "basic" implies there is some other kind of memento out there.